textproc/dblatex/Portfile

```diff
@@ -136,7 +136,7 @@ post-destroot {
     file mkdir ${destroot}${dblatex.texmflocal}/tex/latex/dblatex
     fs-traverse f ${destroot}${python.prefix}/share/dblatex/latex {
         if {[file isfile $f]} {
-            if {[catch {exec ${prefix}/bin/kpsewhich -a [file tail $f]} result]} {
+            if {[catch {exec ${dblatex.mactex_bin}/kpsewhich -a [file tail $f]} result]} {
```

Comment:
Are you sure this is the right fix?  AFAICS, the `fs-traverse` call is outside of the `+mactex` variant code.  If `+mactex` is not used, `dblatex.mactex_bin` is empty...

Reply:
You are right. I'll correct that.